File: Easy/1. Two Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target){
        vector <int> returning;
        for(int i = 0; i < nums.size(); i++){
            for(int j = i+1; j < nums.size(); j++){
                if(nums[i] + nums[j] == target){
                    returning.push_back(i);
                    returning.push_back(j);
                    return returning;
                }
            }
        }
        return returning;
    }
};

File: Easy/1. Two Sum.py
# Naod Philemon
# 12/22/2022

class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        seen = set()

        for num in nums:
            if num in seen:
                return [num, target-num]
            seen.add(target-num)

        return [-1]